packages/extension-chakra-store-locator/src/setup-app.ts

```diff
@@ -71,15 +72,14 @@ class StoreLocatorExtension extends ApplicationExtension<Config> {
         return applyHOCs(App, HOCs)
     }
 
-    extendRoutes(routes: RouteProps[]): RouteProps[] {
-        return [
+    getRoutes(params: GetRoutesParams): Promise<RouteProps[]> {
+        return Promise.resolve([
```

Comment:
@vmarta after discussing with @bendvc  it seems we will go with having a `getRoutes` and `getRoutesAsync` method instead of `getRoutes` that can be either async/sync

In my code I attempted to implement the latter but the only way to check if `getRoutes` is async or not is by calling `getRoutes` and checking if a Promise is returned which Ben pointed out seemed wrong.

in my PR i have a `getRoutes` method that returns `RouteProps[]` and a `getRoutesAsync` that returns a `Promise<RouteProps[]>`. So not sure if we want to apply that in this PR or make that chagne as part of mine
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2300/files#diff-eb67fe9398ed89ecd37d1e431a7599969fe7833eb831b0f0fe226c13b044eec9